types/ember-useragent/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for ember-useragent 0.12
+// Project: https://github.com/willviles/ember-useragent#readme
+// Definitions by: Cameron Dubas <https://github.com/camerondubas>
+//                 Chris Krycho <https://github.com/chriskrycho>
+//                 James C. Davis <https://github.com/jamescdavis>
+//                 Dan Freeman <https://github.com/dfreeman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import './services/user-agent';
+import './helpers/user-agent';
+
+export {};
```

Comment:
what about explicit (and JS native like):
```ts
import UserAgentService from './services/user-agent';

export * from './services/user-agent';
export * from './helpers/user-agent';

declare module '@ember/service' {
    interface ServiceRegistry {
        userAgent: UserAgentService;
    }
}
```
(module declaration is to be removed from `user-agent` btw)

Reply:
Ya I like this. I've pushed an update.